docs/install.rst

```diff
@@ -39,6 +39,14 @@ One easy way to obtain the above is through the AstroConda Python stack:
 
     $ conda create -n astroconda stsci
 
+- Ensure the minimum Python requirements above are met. If needed, specify the 
+minimum version required
```

Comment:
These two lines need to be one - i.e., remove the line break between the two.

Reply:
Alright, I def. made this more complicated than it needed to be... :| Now the LGTM analysis: C/C++ check is failing.

Reply:
You can ignore that.  It seems to occasionally barf.  I just do so.